config/settings/base.py

```diff
@@ -267,6 +268,21 @@ def env(name, default=NoDefaultValue, type_=str):
 USE_TZ = True
 
 
+# Email settings
+DEFAULT_FROM_EMAIL = env("DEFAULT_FROM_EMAIL", default="no-reply@metashare.org")
+SERVER_EMAIL = DEFAULT_FROM_EMAIL
+if DEBUG:
+    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+else:
+    EMAIL_BACKEND = "anymail.backends.mailgun.EmailBackend"
+    ANYMAIL = {
+        "MAILGUN_API_KEY": env("MAILGUN_API_KEY", default=""),
+    }
```

Comment:
For Mailgun we need to be able to set the MAILGUN_SENDER_DOMAIN too: https://github.com/SFDO-Tooling/MetaCI/blob/master/config/settings/production.py#L130